scripts/deploy.ts

```diff
@@ -28,12 +30,18 @@ async function main() {
     let salt = Fr.random();
 
     let schnorrAccount = await getSchnorrAccount(pxe, secretKey, deriveSigningKey(secretKey), salt);
-    const { address, publicKeys, partialAddress } = await schnorrAccount.getCompleteAddress()
-    let tx = await schnorrAccount.deploy().wait();
+    const { address, publicKeys, partialAddress } = await schnorrAccount.getCompleteAddress();
+    await bridgeFeeJuice(pxe, address, FEE_FUNDING_FOR_TESTER_ACCOUNT)
+    const paymentMethod = new FeeJuicePaymentMethod(address);
+
+    await EasyPrivateVotingContract.deploy(wallets[0], address).send().deployed();    
+    await EasyPrivateVotingContract.deploy(wallets[0], address).send().deployed();
+
+    let tx = await schnorrAccount.deploy({fee: {paymentMethod}}).wait();
     let wallet = await schnorrAccount.getWallet();
 
-    const votingContract = await EasyPrivateVotingContract.deploy(wallet, address).send().deployed();
```

Comment:
Use exiting test wallet:
`    const votingContract = await EasyPrivateVotingContract.deploy(wallets[0], wallets[0].getAddress()).send().deployed();`

Reply:
this doesnt resolve how to get an account to pay its own tx fees using fee juice